dominion/dominion_ui/src/components/Hero/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Hero from './HomeHero';
+import background from '../../assets/images/bg/background.png';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing.unit * 4,
+    backgroundImage: `url(${background})`,
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'cover'
+  }
+});
+
+function HomeHero({ classes }) {
+  return (
+    <Grid container item xs={12} className={classes.root}>
+      <Hero />
+    </Grid>
```

Comment:
What does `Grid` actually gives us here as opposed to just returning `<Hero />`?

Reply:
I will clean this up, this was  method phillip was doing i went a different direction and forgot to clean this up.